lib/authority_ecto/changeset.ex

```diff
@@ -7,8 +7,23 @@ defmodule Authority.Ecto.Changeset do
 
   @type field :: atom
 
+  @password_blacklist File.cwd!()
+                      |> Path.join("password_blacklist.txt")
+                      |> File.read!()
+                      |> String.split("\n")
```

Comment:
This file should probably go in `priv`, and I think you can read it this way:

```elixir
@password_blacklist "priv/password_blacklist.txt"
                    |> File.stream!
                    |> Stream.map(&String.trim/1)
                    |> Enum.into([])
```

It's slightly more efficient.